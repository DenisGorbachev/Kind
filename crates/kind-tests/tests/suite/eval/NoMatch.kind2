Main : U60
Main =
    match NoMatch NoMatch.pudding {
        pudding => 2
    }