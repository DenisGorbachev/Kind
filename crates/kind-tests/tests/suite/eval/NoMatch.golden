   ERROR  Required functions are not implemented for this type.

      /--[tests/suite/eval/NoMatch.kind2:3:5]
      |
      |  / 
    3 |  |     match NoMatch NoMatch.pudding {
    4 |  |         pudding => 2
    5 |  |     }
      :  |  
      :  \ You cannot use this expression! 

      Hint: You must implement 'NoMatch.match' in order to use the match notation (or derive match with #derive[match]).

